ramls/oai-pmh-view.raml

```diff
@@ -49,4 +50,25 @@ resourceTypes:
           type: boolean
           required: false
           default: true
-
+  /instances:
+    displayName: Stream API to get instances with items and holdings for OAI-PMH
+    post:
+      description: Stream instances view data for oai-pmh
+      is:
+        [validate]
+      body:
+        application/json:
+          description: List of instances Ids
+          type: string[]
+      queryParameters:
+        skipSuppressedFromDiscoveryRecords:
```

Comment:
Why is this needed in the POST to fetch the specific records? Would the initial query (that determines the instances and remembers the IDs) reflect whether to include or exclude instances that are suppressed from discovery?

Reply:
You are right that skipSuppressedFromDiscoveryRecords from the initial request (updatedInstanceIds) will be used to determine what ids should be passed, but in this query we need to know what items and holdings should be included in the response (based on skipSuppressedFromDiscoveryRecords ). The latest changes passes "skipSuppressedFromDiscovery" in couple with "instance ids" in POST's body payload. And as it was mentioned above it just affect if items/holdings will be excluded from result depending on their "discoverySuppress" value. And such a flag affect th whole bunch of instance ids.

Reply:
Please could the descriptions of both properties make the distinction explicit that this only applies to the holdings and items that are fetched with the instance, rather than the instance itself.

Reply:
Thank you I will fix it.